src/trex_timesync.h

```diff
@@ -98,6 +98,40 @@ class CTimesyncEngine {
     inline void setTimesyncMaster(bool is_master) { m_is_master = is_master; }
     inline bool isTimesyncMaster() { return m_is_master; }
 
+    // define if hardware clock on port `port` is being adjusted with PTP delta
+    inline void setHardwareClockAdjusted(uint16_t port, bool is_hardware_clock_adjusted) {
+        auto iter = m_hta_per_port.find(port);
```

Comment:
some try/catch wouldn't be faster here? I don't how it'd normally done in c++ :(